app/controllers/idv/sessions_controller.rb

```diff
@@ -43,12 +43,10 @@ def clear_session
     end
 
     def log_analytics
-      # LG-7128: Implement Inherited Proofing analytics here.
-      # include InheritedProofingConcern and if inherited_proofing?,
-      # pass flow/param/analytics_ids param to the existing idv_start_over event.
       analytics.idv_start_over(
         step: location_params[:step],
         location: location_params[:location],
+        analytics_id: @analytics_id,
```

Comment:
I don't think that `@analytics_id` is defined in this file anywhere?

Reply:
Ah, maybe not. It was a hasty attempt.  Trying to distinguish Inherited Proofing "start overs" from all others.  Thinking... I may just remove that, then.

Reply:
I was thinking the same.

Reply:
missed a namespace

Reply:
analytics_id is now removed; defer to later implementation.